rawler/data/lenses.toml

```diff
@@ -158,6 +158,13 @@ aperture_range = [[14, 10], [14, 10]]
 
 # Canon EF-S mount lenses
 
+[[lenses]]
+key = "EF-S24mm f/2.8 STM"
+make = "Canon"
+model = "EF-S24mm f/2.8 STM"
+focal_range = [[24, 1], [24, 1]]
+aperture_range = [[28, 10], [22, 1]]
```

Comment:
This should be 28, 10, 28, 10 as only the widest aperture is relevant.

Reply:
fixed